src/components/common/Pagination.vue

```diff
@@ -70,12 +70,10 @@ import { Options, Vue } from "vue-class-component";
             required: true
         },
         previous: {
-            type: Number || undefined,
-            required: true
+            type: Number || undefined
```

Comment:
@sosan-neikbeen vue component prop types aren't working that way. 
https://vuejs.org/guide/components/props.html

Props are optional by default. No need to set ```undefined```.
If the prop is not required, it would be nice to have some default value.

If you need to set multiple types for a props it works like:
```javascript
props: {
    something: {
        type: [String, Number]
    }
}
```